check_ssl_cert

```diff
@@ -4803,6 +4803,8 @@ main() {
             CN_TMP="$(echo "${CN}" | sed "s/|/ PIPE /g")"
 
             if [ -n "${SUBJECT_ALTERNATIVE_NAME}" ]; then
+            
+                CN_TMP="$(echo "${SUBJECT_ALTERNATIVE_NAME}" | sed "s/|/ PIPE /g")"
```

Comment:
Maybe this should just be updated a few lines above?

Reply:
Although a SAN should always be there, there are some certificates which don't have one.

Reply:
Of course I've got a root cert exactly like this! 🤦

Although in testing it I found another bug, fixed in #371 !